source/math/IntBox2D.ooc

```diff
@@ -42,6 +42,7 @@ IntBox2D: cover {
 	bottomCenter ::= IntPoint2D new(this center x, this bottom)
 	empty ::= this size empty
 	init: func@ (=leftTop, =size)
+	init: func@ ~fromFloatBox2D (floatBox2D: FloatBox2D) {this init(floatBox2D left round() as Int, floatBox2D top round() as Int, floatBox2D width round() as Int, floatBox2D height round() as Int)}
```

Comment:
We did not see the function toIntBox2D(). That was the only reason why we added this constructor. 
Shall I take them away again? Or shall I add a corresponding constructor in the FloatBox2D class?